src/index.html

```diff
@@ -5,9 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image-block">
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="Image-product"
+        >
+      </div>
+
+      <div class="card__product-name">
```

Comment:
use semantic tags for text content
```suggestion
      <div class="card__product-name">
```